source/install/self-managed-changelog.md

```diff
@@ -6,11 +6,98 @@ See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release
 
 Latest Mattermost Releases:
 
+- [Release v7.10 - Feature Release](#release-v7-10-feature-release)
 - [Release v7.9 - Feature Release](#release-v7-9-feature-release)
 - [Release v7.8 - Extended Support Release](#release-v7-8-extended-support-release)
-- [Release v7.7 - Feature Release](#release-v7-7-feature-release)
 - [Release v7.1 - Extended Support Release](#release-v7-1-extended-support-release)
 
+## Release v7.10 - [Feature Release](https://docs.mattermost.com/upgrade/release-definitions.html#feature-release)
+
+**Release day: April 14, 2023**
+
+### Important Upgrade Notes
+
+ - In the next release, v7.11, the following repositories will be merged into one: ``mattermost-server``, ``mattermost-webapp``, ``focalboard`` and ``mattermost-plugin-playbooks``. Developers should read the updated [Developer Guide](https://developers.mattermost.com/contribute/developer-setup/) for details. **Playbooks and Boards will be core parts of the product that cannot be disabled**.
+
+**IMPORTANT:** If you upgrade from a release earlier than v7.9, please read the other [Important Upgrade Notes](/upgrade/important-upgrade-notes.html).
+
+### Highlights
+
+#### Boards
+ - 
+
+#### Playbooks
+ - 
+
+#### Calls
+ - 
+
+### Improvements
+
+#### User Interface (UI)
+ - Added the ability to set a reminder to read a post at a specific time via the “More” menu in posts.
+ - Mentions from muted channels are no longer shown or counted on the browser and desktop tabs.
+ - Updated **System Console** descriptions for **Environment > Developer configuration** settings in the **System Console** to clarify that changes require a server restart to take effect.
+ - The custom user status is now shown in the right-hand side and in the **System Console**.
+ - Added the ability to handle multiple emails at once when inviting users.
+ - Added accessibility support to the date picker.
+ - A feedback survey is displayed during a workspace downgrade process from Cloud Professional to Cloud Free.
+ - Migrated the post dot menu to a Material UI (MUI) menu.
```

Comment:
Does this refer to the message (...) **More** option?

Reply:
@AshishDhama Do you know?

Reply:
@cwarnermm Yes more button, I have added images for more clarity.
<img width="1438" alt="Screenshot 2023-04-04 at 1 28 05 AM" src="https://user-images.githubusercontent.com/16203333/229613970-fbfc2210-30dc-49a4-959e-dbfb35b53bd0.png">
<img width="328" alt="Screenshot 2023-04-04 at 1 28 09 AM" src="https://user-images.githubusercontent.com/16203333/229613976-954f177e-bcac-4a09-a3f3-4797e53a5eba.png">